jenkins_pipelines/environments/common/pipeline-mu-aws.groovy

```diff
@@ -0,0 +1,120 @@
+def run(params) {
+
+    timestamps {
+
+        // Environment variables
+        resultdir = "${WORKSPACE}/results"
+        resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
+        // The junit plugin doesn't affect full paths
+        junit_resultdir = "results/${BUILD_NUMBER}/results_junit"
+
+        //Deployment variables
+        deployed_local = false
+        deployed_aws = false
+        local_mirror_params = "--outputdir ${resultdir} --tf susemanager-ci/terracumber_config/tf_files/local_mirror.tf --gitfolder ${resultdir}/sumaform-local"
+        aws_mirror_params = "--outputdir ${resultdir} --tf susemanager-ci/terracumber_config/tf_files/aws_mirror.tf --gitfolder ${resultdir}/sumaform-aws"
+        aws_common_params = "--outputdir ${resultdir} --tf susemanager-ci/terracumber_config/tf_files/SUSEManager-4.1-AWS.tf --gitfolder ${resultdir}/sumaform-aws"
```

Comment:
Use something like ${params.tf_file} but don't use in the common pipeline a concrete terraform file, otherwise we can't reuse this pipeline for other versions.